.rubocop_todo.yml

```diff
@@ -55,6 +75,11 @@ Lint/ShadowingOuterLocalVariable:
     - 'lib/beaker/host_prebuilt_steps.rb'
     - 'lib/beaker/perf.rb'
 
+# Configuration parameters: AllowComments, AllowNil.
+Lint/SuppressedException:
```

Comment:
I think you solved this, or are there more cases of it?

Reply:
done